### PR TITLE
fix screenshot creation logic [UNRELEASED]

### PR DESCRIPTION
**Pull request by samsel**
Signed-off-by: Selvanathan, Sam santhonyselvanat@paypal.com

### CODE REVIEW

**Comment by samsel** [comment]
@nikulkarni @grawk can you guys pull this in?

File: index.js [hunk 1]
```diff
@@ -134,7 +134,8 @@ module.exports = {
                     var imageDir = path.resolve(screenShotPath);
                     var imageFullPath = path.join(imageDir, imageName);
 
-                    mkdirp.sync(imageDir);
+                    // create directories all the way nested down to the last level
+                    mkdirp.sync(imageFullPath.substr(0, imageFullPath.lastIndexOf('/')));
```
Comment (grawk) [inline index.js]: We are baking in windows incompatibility here

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nemo-screenshot",
-  "version": "2.2.0",
+  "version": "2.2.1",
```
Comment (grawk) [inline package.json]: please leave the version unchanged

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # nemo-screenshot changelog
 
+## v2.2.1
```
Comment (grawk) [inline CHANGELOG.md]: please change header name to UNRELEASED (publisher will manage final changelog version

**Review by grawk — COMMENTED** [review]
couple process oriented changes. also raising the issue of windows incompat.

**Comment by samsel** [comment]
@grawk I've made the changes 😄

**Comment by samsel** [comment]
a patch release would be great. our CI is broken because of this!

**Comment by grawk** [comment]
Looks good.

**Comment by grawk** [comment]
published as `v2.2.1`

**Comment by samsel** [comment]
thanks

**Pull request merged**